src/pages/workspace/reportFields/CreateReportFieldsPage.tsx

```diff
@@ -81,7 +81,8 @@ function CreateReportFieldsPage({
                 errors[INPUT_IDS.TYPE] = translate('workspace.reportFields.reportFieldTypeRequiredError');
             }
 
-            if (type === CONST.REPORT_FIELD_TYPES.TEXT && formInitialValue.length > CONST.WORKSPACE_REPORT_FIELD_POLICY_MAX_LENGTH) {
+            // As formInitialValue can be undefined and ts can't identify this yet, we are using !!formInitialValue to check for that.
```

Comment:
```suggestion
            // formInitialValue can be undefined because the InitialValue component is rendered conditionally. If it's not been rendered when the validation is executed, formInitialValue will be undefined.
```